layouts/404-hi.html

```diff
@@ -0,0 +1,28 @@
+{{ define "main" }}
+<div class="container-404">
+    <h4>
+        शायद आप यह देखना चाहते थे:
```

Comment:
In english, it says `Perhaps you were looking for:`

Doesn't the meaning changes to searching for something in Hindi: `शायद आप यह ढूंढ रहे हैं।`

Reply:
I just minded the tenses here, "देखना चाहते थे" for "were looking for". I can change it if you want.

Reply:
@Rajakavitha1 any suggestions on this one.

Reply:
how about: शायद आप यह खोज रहे हैं

Reply:
Sounds good to me, will change in next commit.